lib/tidewave/database_adapters/sequel.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Tidewave
+  module DatabaseAdapters
+    class Sequel < DatabaseAdapter
+      RESULT_LIMIT = 50
+
+      def execute_query(query, arguments = [])
+        db = ::Sequel::Model.db
+
+        # Execute the query with arguments
+        result = if arguments.any?
+          db.fetch(query, *arguments)
+        else
+          db.fetch(query)
+        end
+
+        # Convert to array of hashes and extract metadata
+        rows = result.all
+        columns = rows.first&.keys || []
+
+        # Format the result similar to ActiveRecord
+        {
+          columns: columns.map(&:to_s),
+          rows: rows.first(RESULT_LIMIT).map(&:values),
+          row_count: rows.length,
+          adapter: adapter_name,
+          database: database_name
+        }
+      end
+
+      def get_models
+        # Ensure all models are loaded
+        Rails.application.eager_load!
```

Comment:
We should keep this invocation in the `get_models` tool itself.

Reply:
Actually, nevermind. I will simplify this and return only the models and their location. No need for relationships.